pandas/_libs/tslibs/conversion.pyx

```diff
@@ -90,6 +89,35 @@ cdef class _TSObject:
             return self.value
 
 
+cpdef int64_t pydt_to_i8(object pydt) except? -1:
+    """
```

Comment:
this looks like it has exactly 2 uses, I would much rather
``Timestamp(...).value`` and remove this routine (and remove the tests for it)

```
(pandas) bash-3.2$ grep -r pydt_to_i8 pandas/ --include '*.py' --include '*.pyx'
pandas//_libs/tslib.pyx:cpdef pydt_to_i8(object pydt):
pandas//_libs/tslib.pyx:    Inverse of pydt_to_i8
pandas//_libs/tslibs/offsets.pyx:from pandas._libs.tslib import pydt_to_i8, monthrange
pandas//_libs/tslibs/offsets.pyx:        i8 = pydt_to_i8(dt)
pandas//core/indexes/datetimes.py:    return np.int64(libts.pydt_to_i8(key)).view(_NS_DTYPE)
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected
pandas//tests/scalar/test_timestamp.py:                    assert tslib.pydt_to_i8(result) == expected_tz
pandas//tests/scalar/test_timestamp.py:                    assert tslib.pydt_to_i8(result) == expected_tz
pandas//tests/scalar/test_timestamp.py:                    assert tslib.pydt_to_i8(result) == expected_utc
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected_tz
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected_tz
pandas//tests/scalar/test_timestamp.py:                assert tslib.pydt_to_i8(result) == expected_utc
```